### PR TITLE
Update Manual-Testing.md

### PR DESCRIPTION
**Pull request by Youssef1313**
Addresses https://github.com/dotnet/roslyn/pull/48116#issuecomment-700178264

cc: @jcouv

### CODE REVIEW

File: docs/wiki/Manual-Testing.md [hunk 1]
```diff
@@ -142,7 +143,7 @@ When doing a test pass, copy this page and consider using these status indicator
...
-| | :mag: **Navigate To**<br />- Place caret on a user defined Type reference and press "ctrl + ,"<br />- This should list the User Defined Type in the drop down on the Upper Right corner of the editor and selecting this item will move the cursor to the User Reference Definition<br />- Filters per kind of symbol [Visual Studio 2017]<br />- Results should be sorted as follows: [Visual Studio 2017 version 15.6 Preview 4]<br /><details><summary>Expand for details</summary>TODO</details> | | | |
+| | :mag: **Navigate To**<br />- Place caret on a user defined Type reference and press "ctrl + ,"<br />- This should list the User Defined Type in the drop down on the Upper Right corner of the editor and selecting this item will move the cursor to the User Reference Definition<br />- Filters per kind of symbol [Visual Studio 2017]<br />- Results should be sorted as follows: [Visual Studio 2017 version 15.6 Preview 4]<br /><details><summary>Expand for details</summary>TODO</details><br />- Records support in navigate to | | | |
```
Comment (jcouv) [inline docs/wiki/Manual-Testing.md:146]: Thanks. I'd missed that we already had a section for Navigate To.

**Review by jcouv — APPROVED** [review]

File: docs/wiki/Manual-Testing.md (hunk 1 above)
Comment (Youssef1313) [reply docs/wiki/Manual-Testing.md:146]: @jcouv Given that the linked PR is merged, do we still need this?

Comment (jcouv) [reply docs/wiki/Manual-Testing.md:146]: Nah, all good. I didn't realize we had this line already. Thanks

Comment (Youssef1313) [reply docs/wiki/Manual-Testing.md:146]: > 
> 
> Nah, all good. I didn't realize we had this line already. Thanks

Ok I'll close.

**Pull request closed without merging**